checkov/common/bridgecrew/vulnerability_scanning/integrations/package_scanning.py

```diff
@@ -33,6 +33,15 @@ def _create_report(
             }
             for vul in twistcli_scan_result.get("vulnerabilities") or []
         ]
+        packages = [
+            {
+                "type": package.get("type"),
+                "name": package.get("name"),
+                "version": package.get("version"),
+                "licenses":  package.get("licenses", [])
```

Comment:
does use here ` or []`?

Reply:
@gruebel thanks for the input, `package["licenses"]` might not exist at all so I need a default value.. I could put `None` instead of `[]`, but I'm not sure `or []` is the right approach here. WDYT?

Reply:
it depends what you want to achieve, but I think you should either do
```python
"licenses":  package.get("licenses")
```
or
```python
"licenses":  package.get("licenses") or []
```
then you will send `None` or an empty list to the endpoint.